Services/RatingService.cs

```diff
@@ -0,0 +1,71 @@
+﻿using AonFreelancing.Models;
+using AonFreelancing.Models.Requests;
+using AonFreelancing.Contexts;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Identity;
+using System.Security.Claims;
+using System.Threading.Tasks;
+using System;
+using Microsoft.EntityFrameworkCore;
+
+namespace AonFreelancing.Services
+{
+    public class RatingService
+    {
+        private readonly MainAppContext _mainAppContext;
+
+        public RatingService(MainAppContext mainAppContext)
+        {
+            _mainAppContext = mainAppContext;
+        }
+
+        public async Task<bool> Save(long raterUserId, long ratedUserId, double ratingValue, string comment)
```

Comment:
No need to return a bool value, as this method will either save the entity successfully or throw an exception